app/assets/javascripts/app/components/comment_form/comment_form_ctrl.js

```diff
@@ -0,0 +1,31 @@
+(function() {
+
+  angular
+  .module('council.components')
+  .controller('CommentFormCtrl', CommentFormCtrl);
+
+  function CommentFormCtrl(Comment) {
+    var ctrl = this;
+
+    ctrl.update = update;
+    ctrl.create = create;
+
+    function reset() {
+      ctrl.comment = Comment.createInstance();
+    }
+
+    function create() {
+      ctrl.discussion
```

Comment:
I've been thinking that maybe it makes more sense for a directive's controller to be in the same file as the directive. Even more so if it has `bindToController: true`.

These kind of properties that come from the directive's isolate scope, always make you go back to the directive's file to know what it is.
@ronaldofs @gabrielpoca What do you think?